specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -30,7 +30,7 @@ _Jakarta™ Messaging Specification, Version 3.0_. Available at: _https://jakart
 
 _Jakarta™ Transaction Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/transactions/2.0_
 
-_Java™ Transaction Service, Version 1.0_ (JTS specification)_. Available at: _https://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf_
+_Java™ Transaction Service, Version 1.0 (Draft)_ (JTS specification)_. Available at: _http://download.oracle.com/otn-pub/jcp/7309-jts-1.0-spec-oth-JSpec/jts1_0-spec.pdf_
```

Comment:
I think that this issue was likely already present but I think the "_" after the (Draft) should be removed.

To match Jakarta Transactions (https://github.com/eclipse-ee4j/jta-api/pull/177/files) then it would be something like:

`_Java™ Transaction Service (JTS) Specification, Version 1.0 (Draft)_. Available at: _http://download.oracle.com/otn-pub/jcp/7309-jts-1.0-spec-oth-JSpec/jts1_0-spec.pdf_`

There are some formatting differences and the reference in your file has a TM after Java but in Transactions we have this in a separate file: https://github.com/eclipse-ee4j/jta-api/blame/master/spec/src/main/asciidoc/license-efsl.adoc#L17-L18. I think generally though the change here and in the Jakarta Transactions project would then be aligned.

Perhaps we need the specification change merged first: https://github.com/eclipse-ee4j/jta-api/pull/177

Reply:
The Jakarta Transactions PR is merged. So if you think it's appropriate you could refer to what we are doing: https://github.com/eclipse-ee4j/jta-api/blame/master/spec/src/main/asciidoc/Transactions.adoc#L1390-L1391

Reply:
All of the references in the Platform are italicized (with the underscores), so I'm going to leave them as-is.  I don't think there's a requirement to be consistent across all specs with that convention.  I will move the (JTS) abbreviation earlier in my commit to be consistent.  Otherwise, I think we're good.  Thanks!

Reply:
As I was making my edit, I noticed what you were referring to with the underscore after the (Draft).  Thanks for catching that!